nx.json

```diff
@@ -54,28 +51,13 @@
 			}
 		}
 	},
-	"targetDependencies": {
-		"build": [
-			{
-				"target": "build",
-				"projects": "dependencies"
-			}
+	"namedInputs": {
```

Comment:
what's this new part doing?

Reply:
Good question, `namedInputs` replaces `implicitDependencies` in nx 16: https://nx.dev/deprecated/global-implicit-dependencies